### ISSUE TITLE
distributed model parallelism

### ISSUE THREAD

**Issue by icharlest** [pytorch/pytorch#3857]
Does PyTorch support distributed model parallelism? I divide the model into two parts and run them in different nodes. However, I don't know how to continue the autograd between the two nodes. I think this is a problem about distributed model parallelism. And I didn't find an official or unofficial example show that.

Any one can help me?

**Comment by apaszke** [comment]
Yes, it's mentioned [right at the top of `torch.distributed` docs](http://pytorch.org/docs/master/distributed.html#basics).

We're using GitHub issues for bug reports only, and all questions should be posted on [our forums](discuss.pytorch.org).

**Comment by icharlest** [comment]
Surely? It looks like the distributed data parallelism instead of model parallelism. I read your link, but I cannot find how to realize the distributed model parallelism. Is there an official example for distributed model parallelism?

**Comment by apaszke** [comment]
Sorry I have misread the question, my bad. However, as I said, issues are for bug reports only. Questions should go to the forums. You can post a link to the thread here and I'll answer it.

**Comment by icharlest** [comment]
https://discuss.pytorch.org/t/distributed-model-parallelism/10377

I cannot find this type example. Maybe u can show an example in the official document, I think some people also need this example. Thank you!
